src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/permissions/PermissionsClient.scala

```diff
@@ -1,98 +1,86 @@
 package org.broadinstitute.dsde.agora.server.dataaccess.permissions
 
 import AgoraPermissions._
-import org.broadinstitute.dsde.agora.server.AgoraConfig
+import org.broadinstitute.dsde.agora.server.dataaccess.{ReadAction, ReadWriteAction, WriteAction}
 import org.broadinstitute.dsde.agora.server.exceptions.PermissionNotFoundException
 import org.broadinstitute.dsde.agora.server.model.AgoraEntity
+import slick.jdbc.JdbcProfile
 
-import scala.concurrent.{Await, Future}
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.concurrent.duration._
-import scala.util.Try
+import scala.util.{Failure, Success, Try}
 
-trait PermissionsClient {
-  import AgoraConfig.sqlDatabase.profile.api._
-  val db = AgoraConfig.sqlDatabase.db
+abstract class PermissionsClient(profile: JdbcProfile) {
+  import profile.api._
 
   val timeout = 10.seconds
```

Comment:
I think you were looking at the wrong `timeout`. This is in PermissionsClient. Your commits related to this were in AgoraTestFixture.